components/molecules/offCanvas/components/container.js

```diff
@@ -1,20 +1,61 @@
 import { bool, string } from 'prop-types'
-import styled from 'styled-components'
+import styled, { css } from 'styled-components'
 
 export const OffCanvasDiv = styled.div`
   box-shadow: ${({ show }) => (show ? '-5px 5px 22px -15px rgba(0,0,0,0.75)' : 'none')};
   border-left: ${({ show }) => (show ? 'none' : '1px solid #cecece')};
   position: fixed;
-  top: 0;
-  right: 0;
-  width: ${({ width }) => width};
-  height: 100vh;
   background-color: #fff;
-  transform: ${({ width, show }) => `translateX(${show ? 0 : width})`};
-  transition: transform 375ms cubic-bezier(0.6, -0.28, 0.735, 0.045);
+  transition: all ${({ duration }) => duration}ms ease-in-out;
```

Comment:
I think `all` should be changed to `transform`, to prevent animating other properties, like width getting animated during browser resize which looks weird.

Reply:
@kheruc Good point about scroll disable on overlay. I will add it.
And about `transition`, I'll change It to `transform` to check the result.

Reply:
Its the use of the `all` here thats, change that to `width`